### PR TITLE
2828: format search results

### PR DESCRIPTION
**Pull request by rachaelparris**
Screen Shot 2019-09-06 at 7 59 10 AM

### CODE REVIEW

File: web-client/src/presenter/computeds/advancedSearchHelper.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    const formattedSearchResults = searchResults
+      .sort(compareCasesByDocketNumber)
+      .map(result => {
+        result.contactPrimaryName =
+          result.contactPrimary && result.contactPrimary.name;
+        result.contactSecondaryName =
+          result.contactSecondary && result.contactSecondary.name;
+
+        result.formattedFiledDate = applicationContext
+          .getUtilities()
```
Comment (ericsorenson) [inline web-client/src/presenter/computeds/advancedSearchHelper.js:21]: 💪

**Review by ericsorenson — APPROVED** [review]

File: web-client/src/applicationContext.js [hunk 2]
```diff
@@ -350,14 +350,15 @@ const applicationContext = {
...
     SECTIONS,
     SESSION_DEBOUNCE: 250,
-    SESSION_MODAL_TIMEOUT: 5 * MINUTES, // 5 minutes
+    SESSION_MODAL_TIMEOUT: 5 * MINUTES,
     SESSION_TIMEOUT:
       (process.env.SESSION_TIMEOUT && parseInt(process.env.SESSION_TIMEOUT)) ||
-      55 * MINUTES, // 55 minutes
-    STATUS_TYPES: Case.STATUS_TYPES,
+      55 * MINUTES, // 5 minutes
+    STATUS_TYPES: Case.STATUS_TYPES, // 55 minutes
```
Comment (kkoskelin) [inline web-client/src/applicationContext.js:357]: some of the `// X minutes` comments somehow got shifted in a diff, perhaps? not on the correct lines.

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/applicationContext.js (hunk 2 above)
Comment (rachaelparris) [reply web-client/src/applicationContext.js:357]: I'll fix that in my next PR. 👍

**Pull request merged**